doc/typedoc-sidebar.js

```diff
@@ -0,0 +1,23 @@
+module.exports = [
+  "api/index",
+  {
+    "type": "category",
+    "label": "Classes",
+    "items": [
+      "api/classes/asynciterableiteratortoarraywrapper",
+      "api/classes/replicache",
+      "api/classes/scanresult",
+      "api/classes/transactionclosederror"
+    ]
+  },
+  {
+    "type": "category",
+    "label": "Interfaces",
+    "items": [
+      "api/interfaces/createindexdefinition",
+      "api/interfaces/readtransaction",
+      "api/interfaces/replicacheoptions",
+      "api/interfaces/writetransaction"
+    ]
+  }
+];
```

Comment:
What happened to all the types?

Reply:
They show up in the module page.
![Uploading Screen Shot 2021-04-13 at 10.40.01 AM.png…]()